accountsdb-plugin-postgres/src/postgres_client.rs

```diff
@@ -586,9 +604,11 @@ struct UpdateSlotRequest {
     slot_status: SlotStatus,
 }
 
+#[warn(clippy::large_enum_variant)]
 enum DbWorkItem {
-    UpdateAccount(UpdateAccountRequest),
-    UpdateSlot(UpdateSlotRequest),
+    UpdateAccount(Box<UpdateAccountRequest>),
+    UpdateSlot(Box<UpdateSlotRequest>),
+    LogTransaction(Box<LogTransactionRequest>),
```

Comment:
heh, a bit annoying that we have to use `Box` to get around the size discrepancy issue between `LogTransaction` and the rest of the variants. 

Unfortunately I can't think of a better solution other than maybe separating out the `LogTransaction` into a separate `DbTransactionWorkItem`, and building a separate receive channel.

Reply:
Yes -- the Box was used due to the big size differences of the variants of the enum. I thought about use different send/receivers -- but opted to use this as the small boxes overhead is not worth for the extra complexity.